front-end/src/app/_services/authentication.service.ts

```diff
@@ -10,31 +10,52 @@ export class AuthenticationService {
 
   constructor(private http: HttpClient) { }
 
-  serverAuth(): Observable<any> {
+  serverRegisterAuth(fname: string, lname: string, email: string, password: string): Observable<any> {
 
-    let obj = {
-      token: {
-        id: "1234",
-        accountId: "1"
-      },
+    let reqParams = {
       object: {
-        email: "snovak@test.com",
-        passwordHash: "password",
-        phone: "808-222-2222",
-        background: "I love revature",
-        zipCode: "89521",
-        attachment: ""
+        first_name: fname,
+        last_name: lname,
+        email: email,
+        passwordHash: password
+      }
+    }
+    return this.http.put<any>('http:/localhost:8080/api/register', reqParams)
```

Comment:
might be better to put the "http://localhost:8080" in a service so we don't have to change all the urls in the entire app when we release